src/tests/system/tests/test_memory_cache.py

```diff
@@ -1656,3 +1659,366 @@ def test_memory_cache__truncate__nosigbus(client: Client, ldap: LDAP):
     result = client.tools.id("user-1")
     assert result is not None, "User was not found"
     assert result.user.name == "user-1"
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_memory_cache__verify_timestamp_cache(client: Client, provider: GenericProvider):
+    """
+    :title: Verify the existence of timestamp cache and use lsbsearch on those files
+    :setup:
+        1. Add user
+        2. Start SSSD
+    :steps:
+        1. Execute getent passwd to fetch user details
+        2. Check if timestamps cache file exists
```

Comment:
Existence of 'timestamp cache' is an implementation detail (that might be affected by #7093 btw)

@pbrezina, what is the purpose of 'system tests'? I would expect those to test user visible functionality...